api/api/spec/spec.yaml

```diff
@@ -8264,42 +8410,53 @@ paths:
         '429':
           $ref: '#/components/responses/TooManyRequestsResponse'
 
-  /groups:
-    delete:
+  /agents/insert:
```

Comment:
Some agents endpoints were below the groups endpoints, I moved them above so they are well-organized